frontend/server/src/Controllers/Contest.php

```diff
@@ -3085,7 +3085,24 @@ public static function apiAddProblem(\OmegaUp\Request $r): array {
             )
         );
 
-        return ['status' => 'ok'];
+        $problem = \OmegaUp\DAO\Problems::getByAlias($problemAlias);
+        $solutionStatus = null;
```

Comment:
Que te parece si desde squí asignamos el valor de Solución no encontrada por defecto
```suggestion
        $solutionStatus = \OmegaUp\Controllers\Problem::SOLUTION_NOT_FOUND;
```
?

Así evitas  reasignar el valor a `$solutionStatus` en L3098